src/Hangfire.PostgreSql/PostgreSqlJobQueue.cs

```diff
@@ -95,7 +95,7 @@ public void Enqueue(IDbConnection connection, string queue, string jobId)
       connection.Execute(enqueueJobSql,
         new { JobId = Convert.ToInt64(jobId, CultureInfo.InvariantCulture), Queue = queue });
 
-      if (_storage.Options.EnableLongPolling)
+      if (_storage.Options.EnableLongPolling && SupportsNotifications(connection))
```

Comment:
Hmm... I think we should just throw early, like when initializing queue providers (or even constructor of storage), if user opts to enable long polling when notifications are not supported. We'd avoid similar issues where we check whether polling is enabled, but they are not actually supported.